Resources/template.json

```diff
@@ -0,0 +1,222 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "functionAppPrefix": {
+        "type": "string",
+        "metadata": {
+          "description": "Prefix for the function app resources, in the form dfc-env-proj-fnapp"
+        }
+      },
+      "appServicePlanName": {
+        "type": "string",
+        "metadata": {
+          "description": "App service plan to run the function app under"
+        }
+      },
+      "appServicePlanResourceGroup": {
+        "type": "string",
+        "metadata": {
+          "description": "Resource group the app service plan is in"
+        }
+      },
+      "storageConnectionString": {
+        "type": "string",
```

Comment:
This parameter will contain a secret so they type should be securestring, this will stop its value being written out to logs